cryptus-app/components/wallet_viewer/nft_details/nft_infos/nft_info.module.scss

```diff
@@ -0,0 +1,24 @@
+@use "/styles/variables" as v;
+
+.container {
```

Comment:
Seperate with spaces according to a logic:
    flex: 1 1 auto;
   
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;

    width: 100%;
    margin: 24px 0px 4px 0px;

Reply:
oups
```
flex: 1 1 auto

display: flex;
flex-direction: row;
justify-content: space-between;
align-items: center;

width: 100%;
margin: 24px 0px 4px 0px;
```